osu.Game/osu.Game.csproj

```diff
@@ -32,4 +32,7 @@
     <PackageReference Include="SharpRaven" Version="2.4.0" />
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.6.0" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Overlays\Comments\" />
+  </ItemGroup>
```

Comment:
There doesn't seem to be a need for this to land in the `.csproj`.

Reply:
It was an auto addition, will remove